doc/developer_guide/APIs.rst

```diff
@@ -443,9 +443,36 @@ computation.
 
 A downside of performing redundant computation in the level-1 halo is
```

Comment:
When I build the developer guide using ``make latexpdf`` I get the following warning:

```
checking consistency... ../PSyclone/doc/developer_guide/developers_gocean1p0_stencils.rst: WARNING: document isn't included in any toctree
```

This document is present in the ``doc/developer_guide`` directory but it is not included in the build:
```
processing psyclone-dev.tex... index working_practises psyir psyir_symbols psyir_backends parsing APIs modules transformations psy_data system_specific_setup coding-style zz_bibliography
```

I am aware the problem is not due to this issue, but could you please check what is going on here? Is it intentional for this document to not be included in the developer guide? Thanks!

Reply:
I'm not sure either but there is a comment in the file itself saying that it should be added to the documentation once the dev guide is on master so I suspect it was written before the first version of the dev guide was merged to master. I don't know whether this needs to be added or whether something else has taken its place. Probably an @arporter question.

Reply:
Right, since there is no definite answer on this right now, I suggest we leave it as it is.